package.json

```diff
@@ -6,21 +6,26 @@
   },
   "scripts": {
     "build": "bsb -make-world",
-    "start": "bsb -make-world -w",
-    "clean": "bsb -clean-world"
+    "clean": "bsb -clean-world",
+    "start-bsb": "bsb -make-world -w",
+    "start-reshowcase": "./commands/reshowcase ./example/Demo.bs.js",
+    "start": "npm run clean && concurrently --names 'wp,bsb' -c 'bgBlue.bold,bgGreen.bold' 'npm run start-bsb' 'npm run start-reshowcase'"
```

Comment:
Hum, I feel like we need to run a first pass of `bsb -make-world` before launching `concurrently`, shouldn't we?

Reply:
@bloodyowl It isn't required but it makes an output cleaner and probably prevents `Module not found` errors from appearing if the project is not fully built when webpack started.
I updated the command.